rlclientlib/model_mgmt/restapi_data_transport.cc

```diff
@@ -128,7 +128,7 @@ namespace reinforcement_learning { namespace model_management {
       RETURN_ERROR_LS(_trace, status, exception_during_http_req) << error_code::unknown_s;
     }
 
-    return error_code::success;
+    return request_task.get();

```

Comment:
> return request_task.get(); [](start = 3, length = 27)

Shouldn't this still return error_code such line 115?

Reply:
No.  In this case the behavior is right.  The async task has the error code we are looking for.